diff --git a/node_modules/ffi-napi/deps/libffi/libffi.gyp b/node_modules/ffi-napi/deps/libffi/libffi.gyp
index d02bef7..f15ff38 100644
--- a/node_modules/ffi-napi/deps/libffi/libffi.gyp
+++ b/node_modules/ffi-napi/deps/libffi/libffi.gyp
@@ -74,7 +74,6 @@
               '<(INTERMEDIATE_DIR)/<(RULE_INPUT_ROOT).asm',
             ],
             'action': [
-              'call',
               'preprocess_asm.cmd',
                 'include',
                 'config/<(OS)/<(target_arch)',
